pkg/ccl/crosscluster/logical/logical_replication_job.go

```diff
@@ -532,13 +552,31 @@ func (rh *rowHandler) handleRow(ctx context.Context, row tree.Datums) error {
 					HighWater: &replicatedTime,
 				}
 			}
-			progress.RunningStatus = fmt.Sprintf("logical replication running: %s", replicatedTime.GoTime())
+			progress.RunningStatus = status
+			if fractionCompleted > 0 {
```

Comment:
I think we only want to show the progress bar if 0 < fractionCompleted < 1. Currently, this is overwriting the high watermark, so we will no longer show the high watermark when everything is caught up and advancing.

Reply:
I like that idea

Reply:
if fraction completed is 0, the status is now "all ranges are caught up"